tests/unittests/Foundation/NSAttributedStringTests.mm

```diff
@@ -547,8 +554,10 @@ void assertAttributeAt(
                      options:NSAttributedStringEnumerationReverse
                   usingBlock:^void(NSDictionary* val, NSRange range, BOOL* stop) {
                       ++timesEnumerated;
+                      ASSERT_EQ(0, [val count]);
+                      ASSERT_TRUE(NSEqualRanges(NSMakeRange(1, 3), range));
                   }];
-    ASSERT_EQ(0, timesEnumerated);

```

Comment:
Ah started looking at the tests first

---
In reply to: [73620370](https://github.com/Microsoft/WinObjC/pull/819#discussion_r73620370) [](ancestors = 73620370,73620275)